embulk-docs/src/built-in.rst

```diff
@@ -376,6 +388,23 @@ if you set invalid\_string\_escapes and appear invalid JSON string (such as ``\a
 
 (\*1): Throwing an exception.
 
+The ``columns`` option can extract any value as an Embulk's column.
+
++----------+-----------------------------------------------------------------------------------------------------------+
+| name     | description                                                                                               |
++==========+===========================================================================================================+
+| name     | Name of the column. Select a key within a JSON record or any name can be allowed with ``pointer`` option. |
++----------+-----------------------------------------------------------------------------------------------------------+
+| type     | Type of the column (same as CSV parser's one)                                                             |
++----------+-----------------------------------------------------------------------------------------------------------+
+| pointer  | Set as a JSON Pointer expression if you want to specify any value in a JSON record as the column's value  |
++----------+-----------------------------------------------------------------------------------------------------------+
+| timezone | Set if you want to specify time zone for the timestamp column                                             |
```

Comment:
For a similar reason with above, let's keep only `format` here without `timezone` and `date`.